Library/Formula/mu.rb

```diff
@@ -43,6 +43,10 @@ def install
     # shipped by default with Mac OS X is too old.
     ENV["EMACS"] = "no" if build.without? "emacs"
 
+    # Make --related work on 10.9
+    # https://github.com/djcb/mu/issues/380
+    ENV.append 'CXXFLAGS', '-std=c++11'
```

Comment:
It works on 10.8, and I believe will continue working.

The line can be wrapped in `if version >= 10.9` - I'm not finding what that line will look like in brew.